.gitignore

```diff
@@ -16,6 +16,7 @@
 
 .idea/
 .vscode/
+.DS_Store
```

Comment:
What's the difference between `.DS_Store` and `.DS_Store/`?  @XiShanYongYe-Chang